FrEee/Game/Interfaces/ICommand.cs

```diff
@@ -9,19 +11,19 @@ namespace FrEee.Game.Interfaces
 	/// </summary>
 	public interface ICommand : IPromotable
 	{
-		IReferrable Executor { get; }
+		IReferrable? Executor { get; }
 
-		long ExecutorID { get; }
+		long? ExecutorID { get; }
 
 		/// <summary>
 		/// The empire issuing the command.
 		/// </summary>
-		Empire Issuer { get; }
+		Empire? Issuer { get; }
 
 		/// <summary>
 		/// Any new (from the client) objects referred to by this command.
 		/// </summary>
-		IEnumerable<IReferrable> NewReferrables { get; }
+		IEnumerable<IReferrable?> NewReferrables { get; }

```

Comment:
Hmm. Just at a glance, I don't see why these properties should be nullable. I assume in your first pass you are just marking everything as nullable to get rid of the errors, but later we can go over things and de-nullabilize them as needed?

Reply:
Yes, as a first pass I'm enabling nulls everywhere where it complains - there's still plenty of places! A lot of those should be easy to remove in the second pass (for example, the `GalaxyReference<T>` properties that have a `T` backing field are a bit confusing to the compiler, which doesn't "see" that the backing field is initialized via the reference and assumes that it should be nullable - I don't know yet how to work around it easily, but it should be possible), but given the fact that before _everything_ was nullable anyway, I've assumed that it won't be any worse for the time being.